Source/ResizeableRectItem.cpp

```diff
@@ -399,3 +400,64 @@ void ResizeableRectItem::PosOrRectChanged()
     // Update the property tree view
     mPropSyncer.Sync(this);
 }
+
+void ResizeableRectItem::UpdateIcon()
+{
+    QString images_path = ":/object_images/rsc/object_images/";
+    QString object_name = mMapObject->mObjectStructureType.c_str();
+    
+    if( mTab->GetModel().GetMapInfo().mGame == "AE" )
```

Comment:
I think you can avoid passing the tab and checking if its AE completely here - just check that PropertyByName doesn't return null which you should probably be doing anyway since it will crash if that property is ever renamed in the json

Reply:
Added a change to that in the next PR